pkg/kritis/metadata/containeranalysis/containeranalysis.go

```diff
@@ -75,32 +75,38 @@ func (c Client) Vulnerabilities(containerImage string) ([]metadata.Vulnerability
 			vulnz = append(vulnz, *v)
 		}
 	}
+
 	return vulnz, nil
 }
 
 //Attestations gets AttesationAuthority Occurrences for a specified image.
```

Comment:
Please update the comment to reflect what it does now (also uses the note from the aa).